schema/360-giving-schema.json

```diff
@@ -762,7 +762,7 @@
             "string",
             "null"
           ],
-          "description": "A code referring to a geographical area, drawn from an established gazetteer. For example, the code for a local authority ward, or parliamentary constituency.",
+          "description": "A code referring to a geographical area, drawn from an established gazetteer such as the [ONS Register of Geographic Codes](https://data.gov.uk/dataset/389511fa-eb72-4657-a2ef-b3c982ec4c0e/register-of-geographic-codes-april-2020-for-the-united-kingdom) in the UK. For example, the code for a local authority ward, or parliamentary constituency.",
```

Comment:
Do we know if that link from ONS will persist? As it is time specific I would be more comfortable using a wiki link to generic information - eg https://en.wikipedia.org/wiki/ONS_coding_system

Reply:
I'd second using a Wiki link, I can modify this PR if needed

Reply:
The canonical link for the Register of Geographic Codes would be this one: https://geoportal.statistics.gov.uk/datasets/register-of-geographic-codes-june-2021-for-the-united-kingdom/about - but that changes every time they publish a new one, and the links are likely to change (plus to geoportal is a slightly rubbish js-heavy site so it's never 100% links will work anyway). So wiki probably makes sense.